boards/common/actinius/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022 Actinius
+# SPDX-License-Identifier: Apache-2.0
+
+config ACTINIUS_BOARD_CONTROL_INIT_PRIORITY
+	int "Init priority"
+	default 99
+	help
+	  Sets the init priority for the Board Control module.
+
+config ACTINIUS_BOARD_HAS_SIM_SELECT
+	bool
+	help
+	  Should be selected if an Actinius board provides
+	  a choice for internal or external SIM.
+
+config ACTINIUS_BOARD_HAS_CHARGER_ENABLE
+	bool
+	help
+	  Should be selected if an Actinius board provides
+	  a choice for disabling the charger.
```

Comment:
this is redundant: can be determined using DT API. E.g. `DT_HAS_COMPAT_STATUS_OKAY(actinius_sim_select)`

Reply:
alright, I dropped these two configs, thanks!